src/lib/schedule.c

```diff
@@ -89,3 +89,16 @@ int schedule_sof_task_cancel(struct sof_task *task)
 
 	return ret;
 }
+
+int schedule_sof_init(void)
+{
+	int ret = 0;
+
+	/* init edf scheduler */
+	edf_scheduler_init();
+
+	/* init system llq */
+	init_system_llq();
+
```

Comment:
Let's be somehow consistent. Rename init_system_llq -> system_llq_init();

Although, I would have preferred something like this: 
```
scheduler_init(EDF); scheduler_init(LLQ)
```

but I'm not sure that the init functions can be factored into a single function.

Reply:
@lgirdwood @tlauda 
What's you opinion?